### PR TITLE
[bn] Localize content/bn/docs/concepts/overview/what-is-kubernetes.md

### PR DESCRIPTION
**Pull request by ikramulkayes**
Signed-off-by: Ikramul Kayes [ikramul.kayesgg@gmail.com](mailto:ikramul.kayesgg@gmail.com)
Localizes content/bn/docs/concepts/overview/what-is-kubernetes.md> to Bengali
This is an initial PR as may need some improvements!
Tracking issue: https://github.com/kubernetes/website/issues/31677

### CODE REVIEW

**Comment by mitul3737** [comment]
@ikramulkayes  Please do sign the [EasyCLA.](https://api.easycla.lfx.linuxfoundation.org/v2/repository-provider/github/sign/18706487/51478266/33294/#/?version=2)

**Comment by ikramulkayes** [comment]
Updated my easycla

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+কুবারনেটিস আপনাকে সরবরাহ করে:
+
+* **পরিষেবা আবিষ্কার এবং লোড ব্যালেন্সিং**
+কুবারনেটিস ডিএনএস নাম ব্যবহার করে বা তাদের নিজস্ব আইপি ঠিকানা ব্যবহার করে একটি ধারক প্রকাশ করতে পারে। একটি কন্টেইনারে ট্রাফিক বেশি হলে, কুবারনেটিস লোড ব্যালেন্স এবং নেটওয়ার্ক ট্র্যাফিক বিতরণ করতে সক্ষম হয় যাতে স্থাপনা স্থিতিশীল থাকে।
+* **স্টোরেজ অর্কেস্ট্রেশন**
+কুবারনেটিস আপনাকে স্বয়ংক্রিয়ভাবে আপনার পছন্দের একটি স্টোরেজ সিস্টেম মাউন্ট করার অনুমতি দেয়, যেমন স্থানীয় স্টোরেজ, পাবলিক ক্লাউড প্রদানকারী এবং আরও অনেক কিছু।
+* **স্বয়ংক্রিয় রোলআউট এবং রোলব্যাক**
+আপনি কুবারনেটিস ব্যবহার করে আপনার স্থাপন করা কন্টেইনার জন্য পছন্দসই অবস্থা বর্ণনা করতে পারেন এবং এটি একটি নিয়ন্ত্রিত হারে প্রকৃত অবস্থাকে পছন্দসই অবস্থায় পরিবর্তন করতে পারে। উদাহরণস্বরূপ, আপনি আপনার স্থাপনার জন্য নতুন কন্টেইনার তৈরি করতে, বিদ্যমান কন্টেইনারগুলি সরাতে এবং নতুন কন্টেইনারে তাদের সমস্ত রিসোরস গ্রহণ করতে কুবারনেটসকে স্বয়ংক্রিয়ভাবে করতে পারেন।
```
Comment (mitul3737) [inline content/bn/docs/concepts/overview/what-is-kubernetes.md:67]: Kindly check this sentence `কুবারনেটসকে স্বয়ংক্রিয়ভাবে করতে পারেন।` . Specially the word `রিসোরস` & `কুবারনেটসকে`

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+* **স্বয়ংক্রিয় বিন প্যাকিং**
+আপনি কুবারনেটিসকে নোডের একটি ক্লাস্টার প্রদান করেন যা এটি কন্টেইনারাইজড কাজ চালাতে ব্যবহার করতে পারে। আপনি কুবারনেটসকে বলুন প্রতিটি কন্টেইনারের কত CPU এবং মেমরি (RAM) প্রয়োজন। কুবারনেটিস আপনার সম্পদের সর্বোত্তম ব্যবহার করতে আপনার নোডগুলিতে কন্টেইনারে মানানসই করতে পারে।
```
Comment (mitul3737) [inline content/bn/docs/concepts/overview/what-is-kubernetes.md:69]: `কুবারনেটসকে` should not be here

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+কুবারনেটিস:
+
+* সমর্থিত অ্যাপ্লিকেশনের ধরন সীমাবদ্ধ করে না। কুবারনেটিস-এর লক্ষ্য স্টেটলেস, স্টেটফুল, এবং ডেটা-প্রসেসিং ওয়ার্কলোড সহ অত্যন্ত বৈচিত্র্যময় কাজের লোড সহায়তা করা। যদি একটি অ্যাপ্লিকেশন একটি পাত্রে চলতে পারে তবে এটি কুবারনেটে দুর্দান্ত চালানো সম্ভব।
+* সোর্স কোড স্থাপন করে না এবং আপনার অ্যাপ্লিকেশন তৈরি করে না। ক্রমাগত ইন্টিগ্রেশন, ডেলিভারি, এবং ডিপ্লয়মেন্ট (CI/CD) ওয়ার্কফ্লোগুলি সংস্থার প্রকৃতির উপর এবং পছন্দগুলির পাশাপাশি প্রযুক্তিগত প্রয়োজনীয়তা দ্বারা নির্ধারিত হয়। 
+* মিডলওয়্যার (উদাহরণস্বরূপ, বার্তা বাস), ডেটা-প্রসেসিং ফ্রেমওয়ার্ক (উদাহরণস্বরূপ, স্পার্ক), ডাটাবেস (উদাহরণস্বরূপ, মাইএসকিউএল), ক্যাশে, বা ক্লাস্টার স্টোরেজ সিস্টেম (উদাহরণস্বরূপ, Ceph) এর মতো অ্যাপ্লিকেশন-স্তরের পরিষেবা প্রদান করে না। অন্তর্নির্মিত পরিষেবা হিসাবে। এই ধরনের উপাদানগুলি কুবারনেটিস-এ চলতে পারে, এবং/অথবা পোর্টেবল মেকানিজম, যেমন [ওপেন সার্ভিস ব্রোকার](https://openservicebrokerapi.org/) এর মাধ্যমে Kubernetes-এ চলমান অ্যাপ্লিকেশনের মাধ্যমে অ্যাক্সেস করা যেতে পারে।
```
Comment (mitul3737) [inline content/bn/docs/concepts/overview/what-is-kubernetes.md:83]: You can use `মাইএসকিউএল(MySql)` , `স্পার্ক(Spark)`

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+* লগিং, পর্যবেক্ষণ, বা সতর্কতা সমাধান নির্দেশ করে না। এটি ধারণার প্রমাণ হিসাবে কিছু ইন্টিগ্রেশন এবং মেট্রিক্স সংগ্রহ ও রপ্তানি করার প্রক্রিয়া প্রদান করে।
+* কনফিগারেশন ভাষা/সিস্টেম প্রদান করে না বা বাধ্যতামূলক করে না (উদাহরণস্বরূপ, জসননেট)। এটি একটি ঘোষণামূলক API প্রদান করে যা ঘোষণামূলক স্পেসিফিকেশনের নির্বিচারে ফর্ম দ্বারা লক্ষ্যধারন করা যেতে পারে।
+* কোন ব্যাপক যান্ত্রিক কনফিগারেশন, রক্ষণাবেক্ষণ, ব্যবস্থাপনা, বা স্ব-নিরাময় সিস্টেম প্রদান বা গ্রহণ করে না।
+*উপরন্তু, কুবারনেটিস একটি নিছক অর্কেস্ট্রেশন সিস্টেম নয়। প্রকৃতপক্ষে, এটি অর্কেস্ট্রেশনের প্রয়োজনীয়তা দূর করে। অর্কেস্ট্রেশনের প্রযুক্তিগত সংজ্ঞা হল একটি সংজ্ঞায়িত কর্মপ্রবাহের সঞ্চালন: প্রথমে A, তারপর B, তারপর C করুন। বিপরীতে, Kubernetes স্বাধীন, সংমিশ্রণযোগ্য নিয়ন্ত্রণ প্রক্রিয়াগুলির একটি সেট নিয়ে গঠিত যা বর্তমান অবস্থাকে প্রদত্ত পছন্দসই অবস্থার দিকে ক্রমাগত চালিত করে। আপনি A থেকে C পর্যন্ত কিভাবে যাবেন তা বিবেচ্য নয়। কেন্দ্রীভূত নিয়ন্ত্রণেরও প্রয়োজন নেই। এর ফলে এমন একটি সিস্টেম তৈরি হয় যা ব্যবহার করা সহজ এবং আরও শক্তিশালী, মজবুত, স্থিতিস্থাপক এবং বর্ধনশীল।
+
+
+
+## {{% heading "এরপর কি" %}}
```
Comment (mitul3737) [inline content/bn/docs/concepts/overview/what-is-kubernetes.md:91]: You don't need to localize this one `{{% heading "whatsnext" %}}`

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+*   কটাক্ষপাত করুন [Kubernetes Components](/docs/concepts/overview/components/)
```
Comment (mitul3737) [inline content/bn/docs/concepts/overview/what-is-kubernetes.md:93]: Can you check this sentence? `কটাক্ষপাত করুন [Kubernetes Components]` . The sentence has not been properly localized

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+*   প্রস্তুত [Get Started](/docs/setup/)?
```
Comment (mitul3737) [inline content/bn/docs/concepts/overview/what-is-kubernetes.md:94]: Can you localize this one too properly `প্রস্তুত [Get Started]` ?

**Review by mitul3737 — CHANGES_REQUESTED** [review]

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
+---
+reviewers:
+- bgrant0607
+- mikedanese
+শিরোনাম: কুবারনেটিস কি ?
+বর্ণনা: >
+  কুবারনেটিস হল একটি পোর্টেবল, বর্ধনশীল, ওপেন সোর্স প্ল্যাটফর্ম যা কনটেইনারাইজড ওয়ার্কলোড এবং পরিষেবাগুলি পরিচালনা করার জন্য, যা ঘোষণামূলক কনফিগারেশন এবং অটোমেশন উভয়কেই সহজতর করে। এটির একটি বড়, দ্রুত বর্ধনশীল ইকোসিস্টেম রয়েছে। কুবারনেটিস পরিষেবা, সমর্থন, এবং সরঞ্জাম ব্যাপকভাবে উপলব্ধ।
+বিষয়বস্তুর_প্রকার: ধারণা
```
[the remark this answers is not in the document]
Comment (ikramulkayes) [reply content/bn/docs/concepts/overview/what-is-kubernetes.md:6]: from line 3 to 12 or from line 7 to 12, can you be more specific about which portion of the file should remain in English?

[the remark this answers is not in the document]
Comment (mitul3737) [reply content/bn/docs/concepts/overview/what-is-kubernetes.md:6]: @ikramulkayes from lines 6-12, you don't need to localize these lines.

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by ikramulkayes** [comment]
@mitul3737  check the updates.

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+* **স্বয়ংক্রিয় বিন প্যাকিং**
+আপনি কুবারনেটিসকে নোডের একটি ক্লাস্টার প্রদান করেন যা এটি কন্টেইনারাইজড কাজ চালাতে ব্যবহার করতে পারে। আপনি কুবারনেটিসকেে বলুন প্রতিটি কন্টেইনারের কত CPU এবং মেমরি (RAM) প্রয়োজন। কুবারনেটিস আপনার সম্পদের সর্বোত্তম ব্যবহার করতে আপনার নোডগুলিতে কন্টেইনারে মানানসই করতে পারে।
+* **স্ব-নিরাময়**
+ কন্টেইনারগুলি পুনরায় বন্ধ করে চালু করে যেগুলো ব্যর্থ হয়, কন্টেইনারগুলিকে প্রতিস্থাপন করে, আপনার ব্যবহারকারী-সংজ্ঞায়িত স্বাস্থ্য পরীক্ষায় সাড়া না দেওয়া কন্টেইনারগুলিকে বাতিল করে এবং ক্লায়েন্টদের কাছে সেগুলিকে উপস্থাপন করে না যতক্ষণ না তারা পরিবেশন করার জন্য প্রস্তুত হয়৷ 
+* **গোপন এবং কনফিগারেশন ব্যবস্থাপনা**
+কুবারনেটিস আপনাকে পাসওয়ার্ড, OAuth টোকেন এবং SSH কীগুলির মতো সংবেদনশীল তথ্য সংরক্ষণ এবং পরিচালনা করতে দেয়। আপনি আপনার কন্টেইনারের চিত্রগুলি পুনর্নির্মাণ না করে এবং আপনার স্ট্যাক কনফিগারেশনে গোপনীয়তা প্রকাশ না করেই গোপনীয়তা এবং অ্যাপ্লিকেশন কনফিগারেশন স্থাপন এবং আপডেট করতে পারেন।
+
+## কুবারনেটিস কি না
```
Comment (mitul3737) [inline content/bn/docs/concepts/overview/what-is-kubernetes.md:73]: I think `কুবারনেটিস কি নয়` makes the title more suitable

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+* লগিং, পর্যবেক্ষণ, বা সতর্কতা সমাধান নির্দেশ করে না। এটি ধারণার প্রমাণ হিসাবে কিছু ইন্টিগ্রেশন এবং মেট্রিক্স সংগ্রহ ও রপ্তানি করার প্রক্রিয়া প্রদান করে।
+* কনফিগারেশন ভাষা/সিস্টেম প্রদান করে না বা বাধ্যতামূলক করে না (উদাহরণস্বরূপ, জসননেট)। এটি একটি ঘোষণামূলক API প্রদান করে যা ঘোষণামূলক স্পেসিফিকেশনের নির্বিচারে ফর্ম দ্বারা লক্ষ্যধারন করা যেতে পারে।
+* কোন ব্যাপক যান্ত্রিক কনফিগারেশন, রক্ষণাবেক্ষণ, ব্যবস্থাপনা, বা স্ব-নিরাময় সিস্টেম প্রদান বা গ্রহণ করে না।
+*উপরন্তু, কুবারনেটিস একটি নিছক অর্কেস্ট্রেশন সিস্টেম নয়। প্রকৃতপক্ষে, এটি অর্কেস্ট্রেশনের প্রয়োজনীয়তা দূর করে। অর্কেস্ট্রেশনের প্রযুক্তিগত সংজ্ঞা হল একটি সংজ্ঞায়িত কর্মপ্রবাহের সঞ্চালন: প্রথমে A, তারপর B, তারপর C করুন। বিপরীতে, Kubernetes স্বাধীন, সংমিশ্রণযোগ্য নিয়ন্ত্রণ প্রক্রিয়াগুলির একটি সেট নিয়ে গঠিত যা বর্তমান অবস্থাকে প্রদত্ত পছন্দসই অবস্থার দিকে ক্রমাগত চালিত করে। আপনি A থেকে C পর্যন্ত কিভাবে যাবেন তা বিবেচ্য নয়। কেন্দ্রীভূত নিয়ন্ত্রণেরও প্রয়োজন নেই। এর ফলে এমন একটি সিস্টেম তৈরি হয় যা ব্যবহার করা সহজ এবং আরও শক্তিশালী, মজবুত, স্থিতিস্থাপক এবং বর্ধনশীল।
+
+
+
+## {{% heading "whatsnext" %}}
+
+*   এটি দেখুন [কুবারনেটিস উপাদান](/docs/concepts/overview/components/)
```
Comment (mitul3737) [inline content/bn/docs/concepts/overview/what-is-kubernetes.md:91]: I think `[কুবারনেটিস উপাদানটি] দেখুন` makes it a perfect sentence

File: content/bn/docs/concepts/overview/what-is-kubernetes.md [hunk 10]
```diff
@@ -0,0 +1,92 @@
...
+*   প্রস্তুত [এখানে থেকে শুরু করুন ](/docs/setup/)?
```
Comment (mitul3737) [inline content/bn/docs/concepts/overview/what-is-kubernetes.md:92]: I think this `[শুরু করতে] প্রস্তুত আপনি?` makes more sense

**Review by mitul3737 — CHANGES_REQUESTED** [review]

**Comment by mitul3737** [comment]
@ikramulkayes look at the required changes to be made. Other than this, you have done a great job ☺️

**Comment by ikramulkayes** [comment]
> @mitul3737 check the updates.

 check the updates.

**Review by mitul3737 — APPROVED** [review]
/lgtm

**Pull request merged**